gate-core/src/main/java/com/netflix/spinnaker/gate/services/internal/EchoService.java

```diff
@@ -24,6 +24,7 @@ public interface EchoService {
   ResponseEntity<Void> webhooks(
       @Path("source") String source,
       @Body CloudEvent cdevent,
+      @Header("Ce-Data") String ceDataJsonString,
```

Comment:
SO accepting this for now, but... headers have caused problems on data processing stuff, so long term, ideally from mainstream, we should look at changing this.